posthog/api/exports.py

```diff
@@ -55,28 +55,53 @@ def validate(self, attrs: Dict) -> Dict:
 
         return attrs
 
+    def synthetic_create(self, reason: str, *args: Any, **kwargs: Any) -> ExportedAsset:
+        validated_data = self.validated_data
+        validated_data["team_id"] = self.context["team_id"]
+        validated_data["created_by"] = None
+
+        instance: ExportedAsset = super().create(validated_data)
+
+        self.generate_export_sync(instance)
+
+        instance.refresh_from_db()
+
+        insight_id = instance.insight_id
+        dashboard_id = instance.dashboard_id
+        if insight_id and not dashboard_id:  # we don't log dashboard activity ¯\_(ツ)_/¯
+            try:
+                insight: Insight = Insight.objects.select_related("team__organization").get(id=insight_id)
+                log_activity(
```

Comment:
Where does this activity get logged? I assume it's not in the PostHog dashboard

Reply:
Various resources have an activity log (tends to be called history in the UI) - because this has `scope="Insight",` it will appear in https://app.posthog.com/insights?tab=history

Reply:
TIL